build/index.js

```diff
@@ -173,7 +180,8 @@ async function npmRun(script, pkg) {
 }
 
 async function cleanPackage(pkg) {
-    if (await npmRun("clean", pkg)) return;
+    const cleanCmd = windowsEnvironment ? "win-clean" : "clean";
+    if (await npmRun(cleanCmd, pkg)) return;
```

Comment:
This would mean that every package that wants to customize cleaning has to provide a `win-clean` script in addition? Is there no other way? That seems a bit excessive.

The problem appears to be that we're using `rm -rf` -> Isn't there the `rimraf` package, or some other central approach we could use?

I think we should at least attempt to run `build` if there is no `win-build` -> That way packages that have portable `build` scripts don't need to specify a dedicated Windows one.